console/src/components/Services/Data/TablePermissions/molecules/PermButtonsSection.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useCallback } from 'react';
+import Button from '../../../../Common/Button';
+import {
+  isJsonString,
+  getConfirmation,
+} from '../../../../Common/utils/jsUtils';
+import { FilterState } from '../utils';
+import { showErrorNotification } from '../../../Common/Notification';
+import { permChangePermissions, permChangeTypes } from '../Actions';
+import styles from '../../../../Common/Permissions/PermissionStyles.scss';
+
+interface PermButtonSectionProps {
+  readOnlyMode: string;
+  query: string;
+  localFilterString: FilterState;
+  dispatch: (...p: any) => void;
+  permissionsState: FilterState;
+  permsChanged: string;
+  currQueryPermissions: string;
+}
+
+const PermButtonSection: React.FC<PermButtonSectionProps> = ({
+  readOnlyMode,
+  query,
+  localFilterString,
+  dispatch,
+  permissionsState,
+  permsChanged,
+  currQueryPermissions,
+}) => {
+  if (readOnlyMode) {
+    return null;
+  }
+
+  const dispatchSavePermissions = useCallback(() => {
+    console.error(localFilterString);
```

Comment:
```suggestion
```

Reply:
How did I miss this? :thinking_face:
can we add `no-console` to eslint rules?

Reply:
I thought we have it enabled for TypeScript files, but maybe it's disabled for `console.error` and only works for `console.log`. Feel free to add.